extensions/functions_datetime.yaml

```diff
@@ -688,3 +688,29 @@ scalar_functions:
           - name: origin
             value: time
         return: time
+  -
+    name: "current_date"
+    description: >-
+      Return the current date.
```

Comment:
Is it obvious that this intended to be in UTC?

Unlike timestamp, the date doesn't have enough information to do time zone conversion, so a parameter or separate function would be needed to be able to support both.

Reply:
The options I see are using the locale the server was set up with and UTC.  So perhaps we need an option.  That could also apply to current_timestamp (either to return a timestamp with the timezone of current or UTC).

Reply:
Alternatively if all of the engines just return UTC then we should require that.